ptx/sec_limit_intro.ptx

```diff
@@ -307,7 +306,7 @@
             <!-- START figures/fig_limit1.tex -->
             <image xml:id="img_limit1">
               <latex-image>
-              <![CDATA[
+              <![[CDATA[
```

Comment:
Is it right to insert the extra bracket here? I suspect not. Or, if that is correct, then this block needs an additional close bracket too.

Reply:
No it causes a crash.

Reply:
That was a last-minute typo on my part. I thought I re-ran validation but missed it somehow.
I'll be more careful on the next one!